docs/asciidoc/static/advanced-pipeline.asciidoc

```diff
@@ -116,6 +116,14 @@ filter {
 }
 --------------------------------------------------------------------------------
 
+To set the stdout output to json format:
+[source,json]
+--------------------------------------------------------------------------------
+output {
+    stdout {codec=>rubydebug} 
```

Comment:
I think the issue is that rubydebug is not json, so we should probably not use the word `json`. If we really want to show json, we can use `codec => json_lines` for stdout (json codec doesn't have line terminators).